frontend/src/components/ModelBox/ModelConfigurationSlider.css

```diff
@@ -1,26 +1,32 @@
-.model-config-slider {
-	width: 90%;
-	margin: 0 auto;
-	padding: 0 0 0 0;
+fieldset {
+	border: none;
+	margin: 0;
+	padding: 0;
 	display: flex;
 	flex-direction: column;
-	justify-content: center;
-	align-items: center;
-}
-
-.model-config-info {
-	display: flex;
-	align-items: center;
 }
 
-.model-config-title {
+legend {
```

Comment:
So if the class name for the `fieldset` is "model-slider-fieldset", then this becomes `.model-slider-fieldset legend`, for example.